### DISCUSSION TITLE
V1.83: Can not find the Windows/msi installation package

### DISCUSSION THREAD

**Discussion by Honghe** [VSCodium/vscodium#1670] [General]
May I ask why I can't find the Windows/msi installation package of version 1.83 in the following 56 assets?
![image](https://github.com/VSCodium/vscodium/assets/1092722/18d2aa0e-502c-4554-8466-9e50b0731bc3)

**Comment by daiyam** [comment]
https://github.com/VSCodium/vscodium/issues/1669
